clang/lib/Parse/ParsePragma.cpp

```diff
@@ -1441,7 +1440,7 @@ bool Parser::zOSHandlePragmaHelper(tok::TokenKind PragmaKind) {
     }
 
     PP.Lex(Tok);
-    Actions.ActOnPragmaExport(IdentName, IdentNameLoc);
+    Actions.ActOnPragmaExport(IdentName, IdentNameLoc, getCurScope());
```

Comment:
Can `ActOnPragmaExport` not check the current scope?

Reply:
The current scope is available inside of the function.  However, the current scope inside of the function may not match the scope where the name reference occurred.  In this case the two are the same.  I thought about removing the argument but I noticed that it is common to pass the the current scope for these `ActOnPragma*` functions when the current scope is needed.  I'm leaning towards keeping the argument to be consistent.